### PR TITLE
[Python] Basics Concept: More edits for concept about, exercise  introduction, and concept links

### PR DESCRIPTION
**Pull request by BethanyG**
Added more references to the concept `about.md` and removed them from the exercise `introduction.md`.  Re-arranged and edited the concept `links.JSON`.

### CODE REVIEW

File: languages/python/concepts/basics/about.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
 Python was created by Guido van Rossum and first released in 1991. The [Python Software Foundation](https://www.python.org/psf/) manages and directs resources for Python and CPython development and receives proposals for changes to the language from [members](https://www.python.org/psf/membership/) of the community via [Python Enhancement Proposals or PEPs](https://www.python.org/dev/peps/).
 
-Python puts a strong emphasis on code readability and (_similar to Haskell_) uses [significant indentation](https://docs.python.org/3/reference/lexical_analysis.html#indentation) to denote function, method, and class definitions. The [zen of Python (PEP 20)](https://www.python.org/dev/peps/pep-0020/) lays out additional inspiration and philosophy.
+Python puts a strong emphasis on code readability and (_similar to Haskell_) uses [significant indentation](https://docs.python.org/3/reference/lexical_analysis.html#indentation) to denote function, method, and class definitions. The [zen of Python (PEP 20)](https://www.python.org/dev/peps/pep-0020/) lays out additional philosophy, as does the essay [What is Pythonic?]((https://blog.startifact.com/posts/older/what-is-pythonic.html).
```
Comment (BethanyG) [inline languages/python/concepts/basics/about.md:5]:
```suggestion
Python puts a strong emphasis on code readability and (_similar to Haskell_) uses [significant indentation](https://docs.python.org/3/reference/lexical_analysis.html#indentation) to denote function, method, and class definitions. The [zen of Python (PEP 20)](https://www.python.org/dev/peps/pep-0020/) lays out additional philosophy, as does the essay [What is Pythonic?](https://blog.startifact.com/posts/older/what-is-pythonic.html).
```

File: languages/python/concepts/basics/about.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+[Python](https://docs.python.org/3/) is a [dynamic and strongly](https://stackoverflow.com/questions/11328920/is-python-strongly-typed) typed [object-oriented](https://en.wikipedia.org/wiki/Object-oriented_programming) programming language. It employs both [duck typing](https://en.wikipedia.org/wiki/Duck_typing) and [gradual typing](https://en.wikipedia.org/wiki/Gradual_typing), via [type hints](https://docs.python.org/3/library/typing.html). It supports multiple programming paradigms including both imperative (_object-oriented, procedural_) and declarative (_functional, concurrent_) flavors. But do not be fooled: while programming across paradigms is fully _supported_, [everything in Python is an object](https://docs.python.org/3/reference/datamodel.html).
```
Comment (cmccandless) [inline languages/python/concepts/basics/about.md:1]: There's a lot going on in this line. Should we convert the links to reference-style links to keep the markdown easier to edit?

The content itself seems good to me.

File: languages/python/concepts/basics/about.md [hunk 3]
```diff
@@ -67,35 +69,33 @@ Traceback (most recent call last):
   File "<stdin>", line 1, in <module>
 TypeError: number_to_the_power_of() missing 1 required positional argument: 'number_two'
 
-...
 
 def number_to_the_power_of_default(number_one, number_two=2):
-    '''Returns float or int.
-
+'''Returns float or int.
```
Comment (cmccandless) [inline languages/python/concepts/basics/about.md:74]:
```suggestion
    '''Returns float or int.
```

File: languages/python/concepts/basics/about.md [hunk 4]
```diff
@@ -67,35 +69,33 @@ Traceback (most recent call last):
...
        Takes number_one and raises it to the power of number_two, returning the result.
     '''
 
     return number_one ** number_two
 
 >>> number_to_the_power_of_default(4)
-16
+>>> 16
```
Comment (cmccandless) [inline languages/python/concepts/basics/about.md:81]: Original was correct. Here's a transcript of a test session I did:

```Python
$ python3.8
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> x = 3
>>> x
3
```

```suggestion
16
```

**Review by cmccandless — CHANGES_REQUESTED** [review]
I got thinking: what if we code run `flake8` against the code snippets inside our docs? I did find a project that does this for Typescript "and more"; might be possible to leverage this.

https://github.com/eslint/eslint-plugin-markdown

File: languages/python/concepts/basics/about.md (hunk 2 above)
Comment (BethanyG) [reply languages/python/concepts/basics/about.md:1]: Yeah. That sounds like a much better plan.  In fact, I should go back and do that for all the concept exercises.

Comment (BethanyG) [reply languages/python/concepts/basics/about.md:1]: I think I'd rather merge in these changes and do the links in a different pass, if you don't mind?

**Review by cmccandless — APPROVED** [review]

**Comment by BethanyG** [comment]
I know this seems a bit crazy, but I am going to close this in favor of a new PR.  Things have moved enough that I don't want to deal with the merge.

**Pull request closed without merging**